contrib/build_wine.sh

```diff
@@ -3,7 +3,7 @@
 
 set -ex
 
-PYTHON_VERSION=3.9.7
+PYTHON_VERSION=3.9.19
```

Comment:
https://www.python.org/ftp/python/3.9.19/amd64/ does not exists.
Latest on 3.9 would be https://www.python.org/ftp/python/3.9.13/amd64/

Reply:
Huh, I thought it worked when I did the rc, but maybe the result I used was from a run before this change was made.